### ISSUE TITLE
No entry for "readelf command"

### ISSUE THREAD

**Issue by YoYoYonnY** [valderman/haste-compiler#375]
I just tried to install haste on Windows 10 but got the following error message:
```
haste-cabal.exe: 'C:\Users\Jonne\AppData\Roaming\cabal\bin\hastec.exe' exited
with an error:
hastec.exe: No entry for "readelf command" in
"C:\\Users\\Jonne\\AppData\\Roaming\\haste\\i386-windows-haste-0.5.4-ghc-7.10.3\\settings"
Command 'C:\Users\Jonne\AppData\Roaming\cabal\bin\haste-cabal.exe' failed with error  code 1
```

**Comment by valderman** [comment]
Building Haste on Windows is a bit fiddly; installing [the pre-built, pre-booted binaries](http://haste-lang.org/downloads) is highly recommended for all non-Linux platforms. Did you follow the [building guide](https://github.com/valderman/haste-compiler/blob/master/doc/building.md)?

**Comment by YoYoYonnY** [comment]
@valderman I unsuccesfully tried to install Haste from Hackage and tried building from git aswell. I followed all instructions except for the fact that I used MinGW instead of Cygwin, which might be a problem. I did not know about the existance of pre-built binaries and will definitly check them out.

**Comment by valderman** [comment]
Closing since this seems to be related to an incomplete build environment. Please reopen if the issue persists when following the building guide.

**Comment by YoYoYonnY** [comment]
If I recall correctly, the portable precompiled binaries worked, but nothing else, including the installer or the instructions in the building guide, did (With the installer, Haste showed up, but executing did not work, and I believe I had to edit registers to get it to uninstall). I think I just ended up adding the portable executables to my `PATH`, and haven't had any problems since (I soon after switched Operating System, though).

I guess the message was: Following the building guide for Windows might not be guaranteed to work, and the Windows Installer and `cabal install` do not always work either. I am not sure if this was a one-time problem, but from issues with building and installing other packages, I know that Windows has these problems often. Unfortunately, I didn't find a solution for this particular problem.

By the way, I did try adding a working `readelf.exe` to the `PATH`, even to `C:\Users\Jonne\AppData\Roaming\haste\i386-windows-haste-0.5.4-ghc-7.10.3\settings`, before I reported this issue, but without success.

**Comment by stla** [comment]
Hello @YoYoYonnY 
 It looks like your are the one who replied here: http://www.academytimes.com/solved/2390903961/integrating-haste-into-stack-tool-chain

Didn't you face an issue with `bin-package-db-0.0.0.0` ? (I do, and I'm lost).

**Comment by stla** [comment]
I've found the required module here: http://haskell.inf.elte.hu/docs/7.9.20141202.noWin32/html/libraries/bin-package-db-0.0.0.0/src/GHC-PackageDb.html (this is the only module used by `haste-compiler`).
Not tested to use `haste` yet, but the compilation of `haste-compiler` ran fine.
